src/intervalbridge.jl

```diff
@@ -0,0 +1,54 @@
+# l ≤ ⟨a, x⟩ + α ≤ u
+# is transformed into
+# ⟨a, x⟩ + α ≥ l
+# ⟨a, x⟩ + α ≤ u
+struct SplitIntervalBridge{T}
+    lower::CR{MOI.ScalarAffineFunction{T}, MOI.GreaterThan{T}}
+    upper::CR{MOI.ScalarAffineFunction{T}, MOI.LessThan{T}}
+end
+function SplitIntervalBridge(instance, f::MOI.ScalarAffineFunction, s::MOI.Interval)
+    lower = MOI.addconstraint!(instance, f, MOI.GreaterThan(s.lower))
+    upper = MOI.addconstraint!(instance, f, MOI.LessThan(s.upper))
+    SplitIntervalBridge(lower, upper)
+end
+MOI.get(b::SplitIntervalBridge, ::MOI.NumberOfConstraints) = 0
+MOI.get(b::SplitIntervalBridge{T}, ::MOI.NumberOfConstraints{MOI.ScalarAffineFunction{T}, MOI.LessThan{T}}) where T = 1
+MOI.get(b::SplitIntervalBridge{T}, ::MOI.NumberOfConstraints{MOI.ScalarAffineFunction{T}, MOI.GreaterThan{T}}) where T = 1
+MOI.get(b::SplitIntervalBridge, ::MOI.ListOfConstraintReferences{F, S}) where {F, S} = CR{F, S}[]
+MOI.get(b::SplitIntervalBridge{T}, ::MOI.ListOfConstraintReferences{MOI.ScalarAffineFunction{T}, MOI.GreaterThan{T}}) where {T} = [b.lower]
+MOI.get(b::SplitIntervalBridge{T}, ::MOI.ListOfConstraintReferences{MOI.ScalarAffineFunction{T}, MOI.LessThan{T}}) where {T} = [b.upper]
+
+# References
+MOI.candelete(instance::MOI.AbstractInstance, c::SplitIntervalBridge) = true
+function MOI.delete!(instance::MOI.AbstractInstance, c::SplitIntervalBridge)
+    MOI.delete!(instance, c.lower)
+    MOI.delete!(instance, c.upper)
+end
+
+# Attributes
+MOI.canget(instance::MOI.AbstractInstance, a::MOI.ConstraintPrimal, c::SplitIntervalBridge) = true
```

Comment:
Why does `SplitIntervalBridge` have both constraint attributes and instance attributes?

Reply:
It can act both as a Instance and as a constraint reference. Just like a bridge, it has two sides.